### PR TITLE
[workers] add routing to source-of-truth

### LINKED ISSUE #6339: [Workers] Update wrangler source of truth documentation to include routes
### Which Cloudflare product does this pertain to?

Workers

### Existing documentation URL(s)

https://developers.cloudflare.com/workers/wrangler/configuration/#source-of-truth

### Section that requires update

https://developers.cloudflare.com/workers/wrangler/configuration/#source-of-truth

### What needs to change?

source of truth section should be updated to reflect/capture that routes are subject to the same handling as environment variables with respect to overriding behavior. 

### How should it change?

source of truth section should be updated to reflect/capture that routes are subject to the same handling as environment variables with respect to overriding behavior. 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by helloimalastair**
closes #6339

### CODE REVIEW

File: content/workers/wrangler/configuration.md [hunk 1]
```diff
@@ -639,4 +639,6 @@ It is a recommended best practice to treat `wrangler.toml` as a source of truth
 
 If you change your environment variables in the Cloudflare dashboard, Wrangler will override them the next time you deploy. If you want to disable this behavior, add `keep_vars = true` to your `wrangler.toml`.
 
+Changing routes in your `wrangler.toml` will behave in a similar way, overriding any routes currently deployed. At the moment, there is no way to disable this behaviour.
```
Comment (deadlypants1973) [inline content/workers/wrangler/configuration.md]: Is this sentence saying that route changes to the `wrangler.toml` will override existing routes in the dashboard?

Comment (helloimalastair) [reply content/workers/wrangler/configuration.md]: Yeah. I asked @Skye-31, as I am not as proficient with `wrangler` as they are, and it seems routes will always be overwritten when your `wrangler.toml` changes, even if you have `keep_vars` enabled.

Comment (Skye-31) [reply content/workers/wrangler/configuration.md]: It does, I tested at the time of being asked 🙂

The only exception to this being when no routes/route property is specified - wrangler will not overwrite anything there

Comment (deadlypants1973) [inline content/workers/wrangler/configuration.md]:
```suggestion
If you change your routes in the dashboard, Wrangler will override them in the next deploy with the routes you have set in your `wranglr.toml`. Currently, there is no way to disable this behaviour.
```

**Review by deadlypants1973 — APPROVED** [review]

File: content/workers/wrangler/configuration.md (hunk 1 above)
Comment (ajadams-ml) [reply content/workers/wrangler/configuration.md]: Thanks @Skye-31 for pointing that out. It saved me a huge headache

**Pull request merged**